core/object/script_language.h

```diff
@@ -152,12 +152,16 @@ class Script : public Resource {
 	virtual void get_script_method_list(List<MethodInfo> *p_list) const = 0;
 	virtual void get_script_property_list(List<PropertyInfo> *p_list) const = 0;
 
-	virtual int get_member_line(const StringName &p_member) const { return -1; }
+	virtual int get_member_line(const StringName &p_member) const {
```

Comment:
This was wrongly changed by clang-format 14 (buggy version), should be changed back.